projects/tl-elements/src/lib/tl-header/tl-header.component.html

```diff
@@ -83,6 +83,19 @@
             <wvr-text-component value=" / 979-845-1951"></wvr-text-component>
           </p>
         </wvre-dropdown-menu-item>
+        <wvre-dropdown-menu-item>
+          <div class="libraries-link">
+            <a href="//library.tamu.edu/galveston/index.php" title="Jack K. Williams Library - Galveston">
```

Comment:
Other lines similar to this have a 'Go to' prefix and a 'website' suffix.

As I don't know what card/requirements are for this - as none are linked - I cannot verify if this is needed or not for this item.